gui/qt/main_window.py

```diff
@@ -3448,7 +3450,14 @@ def on_be(x):
         block_ex_combo.currentIndexChanged.connect(on_be)
         gui_widgets.append((block_ex_label, block_ex_combo))
 
-        system_cameras = QCameraInfo.availableCameras()
+        try:
+            from PyQt5.QtMultimedia import QCameraInfo
+            system_cameras = QCameraInfo.availableCameras()
+        except ModuleNotFoundError as e:
+            QMessageBox.information(self, "Camera disabled",
```

Comment:
Use `self.show_error()`  here which handles the `top_level_window` craziness properly, rather than `QMessageBox.information`....

Additionally, I request that you only show this warning if they mess with the setting (eg on combobox change) -- not show it unconditionally every time the preference pops up... as it could get annoying.

Reply:
Update: You may  also want to gray out the combobox here and add a toolTip() explaining that their Qt is old, etc.